Cairo Desktop/CairoDesktop.Localization/Language.cs

```diff
@@ -486,6 +504,17 @@ static class Language
             { "sSettings_Taskbar_DisplayModeOverlap", "Barra de tarefas sobrepõe janelas"},
             { "sSettings_Taskbar_DisplayModeAutoHide", "Barra de tarefas oculta automaticamente"},
             { "sSettings_Taskbar_EnableTaskbarMultiMon", "Mostrar barra de tarefas em vários monitores"},
+            { "sSettings_Taskbar_FullWidthTaskBar", "Usar a largura total da tela para a barra de tarefas"}
```

Comment:
This line is missing a comma, could you please fix that?

Reply:
There, i think it should be fixed now, sorry about that.